lib/main.js

```diff
@@ -4,7 +4,10 @@ const SinkFs = require('asset-pipe-sink-fs');
 const reader = require('asset-pipe-js-reader');
 const body = require('body/json');
 const express = require('express');
+const { validationError: ValidationError } = require('error-obj');
```

Comment:
At this point I would prefer if we switch to using [boom](https://github.com/hapijs/boom).